pkg/kubectl/cmd/util/editor/editoptions.go

```diff
@@ -326,6 +329,31 @@ func (o *EditOptions) Run() error {
 			return err
 		}
 		return editFn(infos)
+	case ApplyEditMode:
+		infos, err := o.OriginalResult.Infos()
+		if err != nil {
+			return err
+		}
+		var annotationInfos []*resource.Info
+		for i := range infos {
+			data, err := kubectl.GetOriginalConfiguration(infos[i].Mapping, infos[i].Object)
+			if err != nil {
+				return err
+			}
+			if data == nil {
+				continue
+			}
+
+			tempInfos, err := o.updatedResultGetter(data).Infos()
+			if err != nil {
+				return err
+			}
+			annotationInfos = append(annotationInfos, tempInfos[0])
+		}
+		if len(annotationInfos) == 0 {
+			return errors.New("no last-applied-configuration annotation found on resources, to create the annotation, use command `kubectl apply set-last-applied`")
```

Comment:
Sorry to let you update this again.
`kubectl apply set-last-applied --create-annotation` to be more accurate, right?